### PR TITLE
fix: Made URL into a URL box and changed the "actions icons" from vertical to horizontal

### LINKED ISSUE #6472: Discount and Access Code: Make URL into a URL box and change the "actions icons" from vertical to horizontal
To improve the access and discount code overview please change the following:
1. Make the URL into a URL box like on the video area 
2. Change the "actions icons" from vertical to horizontal.

Screenshot from 2021-01-23 23-52-13

### PR DESCRIPTION
**Pull request by afzal442**
Fixes #6472 

#### Short description of what this resolves:
SS After
c3

#### Changes proposed in this pull request:

- Made URL box and changed cell actions
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
What is this?

**Comment by afzal442** [comment]
Sir tell me if I am missing something.

**Comment by iamareebjamal** [comment]
Everything

**Comment by iamareebjamal** [comment]
Please read the issue carefully for 5 minutes before working on it

**Comment by afzal442** [comment]
@iamareebjamal sir have a look it again.

**Comment by Creatoon** [comment]
@afzal442 url box looks like this

Screenshot_2021-01-24 Live Events eventyay

**Comment by afzal442** [comment]
> @afzal442 url box looks like this
> 
> Screenshot_2021-01-24 Live Events eventyay
> 

R u talking about video source URL or join video?? Thanks.

**Comment by afzal442** [comment]
> @afzal442 url box looks like this
> 
> Screenshot_2021-01-24 Live Events eventyay
> 

Wher did u get this?? Plz tell

**Comment by Creatoon** [comment]
> R u talking about video source URL or join video?? Thanks.

@afzal442 join video

File: app/templates/components/ui-table/cell/events/view/tickets/access-codes/cell-url.hbs [hunk 1]
```diff
@@ -1 +1,18 @@
-<a href='{{this.record}}' target='_blank' rel='noopener'> {{this.record}} </a>
+<div class="ui action input" style="width: 100%; height: 36px">
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/events/view/tickets/access-codes/cell-url.hbs:1]: Why

File: app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs [hunk 2]
```diff
@@ -1 +1,16 @@
-<a href='{{this.record}}' target='_blank' rel='noopener'> {{this.record}} </a>
+<div class="ui action input" style="width: 100%; height: 36px">
```
Comment (iamareebjamal) [inline app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs:1]: Why

File: app/templates/components/ui-table/cell/events/view/tickets/access-codes/cell-url.hbs (hunk 1 above)
Comment (afzal442) [reply app/templates/components/ui-table/cell/events/view/tickets/access-codes/cell-url.hbs:1]: Sir should I use class or what?

File: app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs (hunk 2 above)
Comment (afzal442) [reply app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs:1]: In the same way it is in the cell video URL.

Comment (iamareebjamal) [reply app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-url.hbs:1]: Use URLBox component

**Comment by iamareebjamal** [comment]
Not fixed. 

You don't read the issue carefully, you don't know the end result. Thus, so much time is wasted explaining what is needed. Hence, from now on, if you want to work on an issue, before raising a PR, explain in the issue comment what do you think is needed in the issue and send a screenshot of fixed issue BEFORE raising the PR

Secondly, you don't understand what is asked in review to be changed, I asked you to use URLBox, but I don't even know what you changed and called it a day. It seems you don't know the general technical terms and lingos in context of this project, or even general ember or any other frontend framework

That's why I recommended you previously to first learn these things and then contribute. Trying to make you understand the issue itself is twice as more time consuming than solving the issue myself. And then there is a barrage of request review cycles where I would ask you to do something and you will randomly change something across 15 commits and nothing will come out of it. 

And while this happens, other contributors who could solve it in 5 minutes are sitting idly and issue is not solved for a week. Beginners taking time is not an issue, but when it is the same story with every PR and the person is not improving the time taken to solve the issue, in fact, making even more random changes like adding a border to input box when it is CLEARLY mentioned in the issue to use URLBox like in Video section, then it is extremely frustrating. 

You expect maintainers to do the work for you even when issues are fairly unambiguous. You think I'll just make some random PR and maintainers will anyway have to tell what's wrong and after 36 moon cycles, my PR will be ready. 

There is a severe communication gap and I don't know how to fix it. Thus, don't raise a PR without commenting the screenshot of the fix in the issue. We simply don't have that much time to go through this trivial stuff

**Comment by afzal442** [comment]
Thanks for your suggestions. There is no problem if the issue is simple and unambiguous and not solved within short period of time, u can close it the same day or next day so that someone else can try out. See there are still so many PRs hanging since many days. 
There is no need to explain such things. This platform is for collaborative team work, but at the same time it is contest for us.

**Comment by iamareebjamal** [comment]
All of those PRs are 50% 70% 80% complete and when they started, they had an idea about what to solve and after reviews, they actually changed what they were supposed to

**Comment by afzal442** [comment]
Sir the same way when I was solving linking names and photo, I discovered the files first where to use linkto route which was an idea to solve then I stuck on models, I asked u in meeting about that issue, u suggested me to change then I found in console, it worked. Then I resolved 80% of that task with figuring out linkto, href using guides + exploring other files. Thanks.

**Pull request closed without merging**